antenna-workflow-steps/processors/enricher/antenna-p2-resolver/src/main/java/org/eclipse/sw360/antenna/p2resolver/P2RepositoryExtractor.java

```diff
@@ -28,6 +29,15 @@ private P2RepositoryExtractor() {
         // Utility class
     }
 
+    public static void installEclipseProductForP2Resolution(String extractionLocation) throws AntennaException {
+        String location = P2RepositoryExtractor.class.getProtectionDomain().getCodeSource().getLocation().getPath();
+        try (JarFile jar = new JarFile(location)) {
+            P2RepositoryExtractor.extractProductFromJar(extractionLocation, location);
+        } catch (IOException e) {
+            P2RepositoryExtractor.extractProductFromFilesystem(extractionLocation, location);
```

Comment:
This exception handling only works for tests.

Reply:
Added a log comment, I think this should be enough.